### PR TITLE
fixes #10437 - remove incorrect/duplicate hash statements

### PR DESCRIPTION
**Pull request by ohadlevy**

### CODE REVIEW

**Comment by iNecas** [comment]
:+1:

File: app/controllers/api/v1/media_controller.rb [hunk 1]
```diff
@@ -25,7 +25,7 @@ class MediaController < V1::BaseController
 
       api :GET, "/media/", "List all media."
       param :search, String, :desc => "filter results", :required => false
-      param :order, String, :desc => "sort results", :required => false, :desc => "for example, name ASC, or name DESC"
+      param :order, String, :required => false, :desc => "for example, name ASC, or name DESC"
```
Comment (domcleal) [inline app/controllers/api/v1/media_controller.rb]: "sort results" is the usual description, please delete the other one.  ditto below

**Comment by ohadlevy** [comment]
@domcleal updated.

**Comment by dLobatog** [comment]
Merged as d9bc6d9cbfa11a2df6ab70973776174d37426adf, thanks @ohadlevy!

**Pull request closed without merging**